src/components/pages/get-help/faq/faq.jsx

```diff
@@ -40,24 +40,44 @@ const items = [
   {
     question: 'Where can I find Cilium tutorial or workshop?',
     answer:
-      'Get hands-on experience with Cilium through interactive courses, tutorials, and official documentation. Practice using Cilium’s networking, observability, and security features in labs provided by companies within the Cilium ecosystem. Check out the <a href="https://cilium.io/labs/" target="_blank" rel="noopener noreferrer" class="text-blue-600 underline">Cilium Labs</a> to get started.',
+      'Get hands-on experience with Cilium through interactive courses, tutorials, and official documentation. Practice using Cilium\'s networking, observability, and security features in labs provided by companies within the Cilium ecosystem. Check out the <a href="https://cilium.io/labs/" target="_blank" rel="noopener noreferrer" class="text-blue-600 underline">Cilium Labs</a> to get started.',
   },
 ];
 
-const Faq = () => (
-  <section className="bg-white py-10 md:py-16 lg:py-20 mt-10 md:mt-20 lg:mt-28 xl:mt-40">
-    <Container size="xs">
-      <Heading className="text-center" size="lg" tag="h2">
-        {title}
-      </Heading>
-      <dl className="mt-6 md:mt-10 lg:mt-14">
-        {items.map((item, index) => {
-          const faqId = `faq${index + 1}`;
-          return <Item {...item} faqId={faqId} key={index} isDefaultOpen={index === 0} />;
-        })}
-      </dl>
-    </Container>
-  </section>
-);
+const Faq = () => {
+  // Generate FAQ Schema as JSON-LD
+  const faqSchema = {
+    '@context': 'https://schema.org',
+    '@type': 'FAQPage',
+    mainEntity: items.map((item) => ({
+      '@type': 'Question',
+      name: item.question,
+      acceptedAnswer: {
+        '@type': 'Answer',
```

Comment:
Google's FAQ schema actually supports valid HTML tags (like < h1 >) inside the answer property.

Since our FAQ content (line 43) includes anchor tags pointing to helpful resources, we should remove .replace(/<[^>]*>/g, '') here. Keeping the HTML allows users to click links directly from the Google search results page, which is a better user experience and can improve CTR.